### ISSUE TITLE
Commands succeed, but sometimes print error LIBUSB_ERROR_IO

### ISSUE THREAD

**Issue by orangejulius** [pjanouch/sensei-raw-ctl#6]
First of all, thanks for making this, it works great!

The only bug I notice is that more often than not, an error is printed when making any sort of changes. I can't figure out a pattern to it, but all the commands below do in fact work (the backlight changes correctly, etc).

I'm quite familiar with C, cmake, and linux, but not libusb. I'm happy to keep digging, but would love some suggestions on how to track down the cause.

```
spectre256@laguna ~/repos/sensei-raw-ctl $ ./sensei-raw-ctl --show
Backlight intensity: off
Backlight pulsation: medium
Speed in CPI (LED is off): 1620
Speed in CPI (LED is on): 3240
Polling frequency: 500Hz
spectre256@laguna ~/repos/sensei-raw-ctl $ ./sensei-raw-ctl --save
spectre256@laguna ~/repos/sensei-raw-ctl $ ./sensei-raw-ctl --intensity high
spectre256@laguna ~/repos/sensei-raw-ctl $ ./sensei-raw-ctl --intensity low
spectre256@laguna ~/repos/sensei-raw-ctl $ ./sensei-raw-ctl --intensity medium
Error: operation failed: LIBUSB_ERROR_IO
spectre256@laguna ~/repos/sensei-raw-ctl $ ./sensei-raw-ctl --intensity low
Error: operation failed: LIBUSB_ERROR_IO
spectre256@laguna ~/repos/sensei-raw-ctl $ ./sensei-raw-ctl --intensity high
spectre256@laguna ~/repos/sensei-raw-ctl $ ./sensei-raw-ctl --intensity off
Error: operation failed: LIBUSB_ERROR_IO
spectre256@laguna ~/repos/sensei-raw-ctl $ ./sensei-raw-ctl --intensity high
spectre256@laguna ~/repos/sensei-raw-ctl $
```

**Comment by pjanouch** [comment]
I know. :confused: Also see d034b7bb2a396207b2656b55a89e905a97557801.

Personally I get a lot of ```LIBUSB_ERROR_PIPE```, which is supposed to happen ```if the control request was not supported by the device```. Only it occurs seemingly at random.

It's kind of hard to track these down and I don't really feel like putting much effort into this, but you're very free to come up with your own patches. All I can think of at the moment is repeating the operation a certain amount of times with a timeout.

As for libusb, basically you're just sending the device some packets. Specifically we're "replaying" packets sent by the Windows driver. The libusb API is not that complicated.

**Comment by pjanouch** [comment]
I believe this has been fixed with 72fb336 and now I feel stupid.
